### ISSUE TITLE
Voiceover: Autocomplete not reading option list

### ISSUE THREAD

**Issue by sekhar-shashi** [angular/components#3544]
https://material.angular.io/components/component/autocomplete
I tried the auto complete component listed above, but once the drop down opens up , with  keyboard up/down the selected element is not readable in screenreader i.e element is not accessible.

#### expected behaviour?
Material Component - Autocomplete, should be accessible/support WCAG (on key up/down/tab/enter...)

#### current behaviour?
Once we type some letter the dropdown populates elements , and when user tries to use arrow key it should be able to read item /accessible(CMD+F5 --> to enable screen reader in MAC).
So currently its not supporting all WCAG features.

#### What are the steps to reproduce?
1) Go to link https://material.angular.io/components/component/autocomplete
2)Click example tab
3) Enable screen reader options (CMD + F5 shortcut in mac).
4)use tab and navigate to input box: state and press enter.
5) it opens up the dropdown with list of states .
6) on key down/up the elements are not accessible .

#### What is the use-case or motivation for changing an existing behaviour?
If we try :https://material.angularjs.org/latest/demo/autocomplete
the basic usage section(which again display states ) is accessible. 

#### Which versions of Angular, Material, OS, browsers are affected?
Latest

**Comment by kara** [comment]
Hm, JAWS and Voiceover appear to need different things. Looking into it.

**Comment by DaveBest99** [comment]
A quick test using Voiceover on iPhone iOS10 shows that the State Autocomplete element is not accessible to touch gestures. However, I can make it work somewhat using the following steps:
1. Single finger swipe right until you hear "State state combo box collapsed". Note, the text label is spoken twice.
2. Single finger double tap on the State Combo box element. Voiceover says "State combo box expanded". Note, if you perform any touch gesture like single finger swipe, the Combo box will close automatically.
3. Expected behaviour would be to single finger swipe right and left to go through the list of States. However, the user only hears the page elements and no Combo box content displayed, although it is expanded.
4. Try single finger double tap and hold on the second tap. This may expose the list items, but the rest is guess work, as touching the screen in the correct spot may select a State in the list (not a good solution). I was able to select Washington, but it took several tries.

**Comment by mahmood-sajjadi** [comment]
Is there any update on this ticket? from @kara  or anyone else?
if there is any progress would you please share the tag/branch then I can work on it too?

thanks

**Comment by mahmood-sajjadi** [comment]
Hi again

autocomplete in MaterialV1 AngularJS(V1) with voice over working as expected
because:
there is an element:
`<div class="md-visually-hidden" role="status" aria-atomic="true" aria-live="assertive">[SOME TEXT]</div>`

which contain something like:
There are 50 matches available.
Alaska
etc.

but we don't have this element is MaterialV2

is there any reason that this element is removed?
I believe adding this element will solve Voiceover issue. and I am happy to do that

my main question is: is there any reason that this element is removed from MaterialV2?

**Comment by MahamBoghani** [comment]
Hi! Any update on this issue? I'm running into this as well.

**Comment by ecerta** [comment]
We've run into it as well. Any update?

**Comment by jelbourn** [comment]
I believe this has been resolved in more recent versions of Angular. In particular, #20819 ensured that options are always read out.
